root/etc/e-smith/templates/etc/httpd/conf.d/nethserver.conf/30include

```diff
@@ -1,8 +0,0 @@
-#
-# 30include -- old ns6 vhost definitions
-#
-
-IncludeOptional nethserver.d/*.vhost
```

Comment:
A simple command that restores ns6 configuration could be

```
cat /etc/httpd/nethserver.d/*.vhost > /etc/httpd/conf.d/ns6-migration.conf
```

Then, as usual

* reload Apache
* add ``ns6-migration.conf`` to backup